Source/Evolve/WarpXEvolve.cpp

```diff
@@ -165,6 +165,7 @@ WarpX::Evolve (int numsteps)
         if (do_electrostatic != ElectrostaticSolverAlgo::None)
         {
             const bool skip_deposition = true;
+            if (warpx_py_particleinjection) warpx_py_particleinjection();
```

Comment:
I believe this should now be removed since I think it is duplicated a few lines earlier.